libs/api/domains/auth/src/lib/models/delegation.model.ts

```diff
@@ -24,9 +24,15 @@ registerEnumType(DelegationType, { name: 'AuthDelegationType' })
         return LegalGuardianDelegation
       case DelegationType.ProcurationHolder:
         return ProcuringHolderDelegation
+      case DelegationType.PersonalRepresentative:
+        return PersonalRepresentativeDelegation
       case DelegationType.Custom:
         return CustomDelegation
     }
+    const exhaustiveCheck = (param: never) => {
+      throw new Error(`Missing interfaceType ${param}`)
```

Comment:
More readable to have this as the default statement of switch.

This error will show up at build time, right?

Reply:
Yes, this will cause a compile error if not all cases are covered. The error thrown should never happen unless `Delegation.Type` would be changed to something like `any`, in fact I believe it is removed from the resulting javascript.

Regarding setting it as default case, the linter will complain `Unexpected lexical declaration in case block.` since I'm declaring a const there. I could disable it. What way to do we prefer?